editoast/editoast_models/src/rolling_stock/rolling_stock_category.rs

```diff
@@ -0,0 +1,84 @@
+use std::io::Write;
+use std::str::FromStr;
+
+use diesel::deserialize::FromSql;
+use diesel::deserialize::FromSqlRow;
+use diesel::expression::AsExpression;
+use diesel::pg::Pg;
+use diesel::pg::PgValue;
+use diesel::serialize::Output;
+use diesel::serialize::ToSql;
+use serde::Deserialize;
+use serde::Serialize;
+use strum::Display;
+use strum::EnumString;
+use strum::IntoStaticStr;
+use utoipa::ToSchema;
+
+editoast_common::schemas! {
+    RollingStockCategory,
+    RollingStockCategories,
+}
+
+/// This enum maps to a Postgres enum type, specifically `rolling_stock_category`.
+/// Any changes made to this enum must be reflected in the corresponding Postgres enum,
+/// and vice versa, to ensure consistency between the application and the database.
```

Comment:
```suggestion
// This enum maps to a Postgres enum type, specifically `rolling_stock_category`.
// Any changes made to this enum must be reflected in the corresponding Postgres enum,
// and vice versa, to ensure consistency between the application and the database.
```

It's not really a doc comment, and doing so makes it appear in the OpenAPI

Reply:
Done.